alert.html

```diff
@@ -0,0 +1,44 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vui-colors/colors.html">
+
+<dom-module id="vui-alert">
+    <template>
+        <style>
+            :root {
+                --very-red: #ff7962;
```

Comment:
Cool, yeah we took the style from the existing non-Daylight alert. We figured we'd get this to web components in the first PR and then start Daylight'ing after... make sense?